src/schema/metadata/ElectricalStimulation.yaml

```
@@ -0,0 +1,7 @@
+---
+name: ElectricalStimulation
+description: |
+  Boolean field to specify if electrical stimulation was done during the
+  recording (options are "true" or "false"). Parameters for event-like
+  stimulation should be specified in the events.tsv file (see example below).
```

it "see example below" still meaningful in this schema form?

---

Good point! I'll remove.

```suggestion
  stimulation should be specified in the events.tsv file.
```